cities/models.py

```diff
@@ -0,0 +1,14 @@
+from sqlalchemy import Column, Integer, String
+from sqlalchemy.orm import relationship
+
+from database import Base
+
+
+class City(Base):
+    __tablename__ = "cities"
+
+    id = Column(Integer, primary_key=True, index=True)
+    name = Column(String(255))
+    additional_info = Column(String, nullable=True)
+
+    temperatures = relationship("Temperature", back_populates="city")
```

Comment:
Ensure that the `Temperature` model is correctly defined and includes a `city` attribute for the `back_populates` relationship to work. This relationship assumes that the `Temperature` model has a `city` attribute that refers back to the `City` model.